### ISSUE TITLE
Missing container?

### ISSUE THREAD

**Issue by gdsmith** [confluentinc/kafka-streams-examples#50]
I get the following error when trying to run `docker-compose up -d`

```
Pulling kafka-music-data-generator (confluentinc/kafka-streams-examples:4.0.0-SNAPSHOT)...
ERROR: manifest for confluentinc/kafka-streams-examples:4.0.0-SNAPSHOT not found
```

**Comment by netrounds-joakim** [comment]
I get the same error

**Comment by netrounds-joakim** [comment]
This was fixed in #66
